src/Symfony/Component/Uid/Uuid.php

```diff
@@ -100,27 +95,23 @@ public function getType(): int
         return uuid_type($this->uuid);
     }
 
-    public function getVariant(): int
-    {
-        return uuid_variant($this->uuid);
-    }
-
-    public function getTime(): int
+    public function getTime(): float
```

Comment:
I don't get why `uuid_time` is limited to an integer timestamp while the UUID time is a 60bit number. Let's return a float instead, with subsecond info.